Error while deploying Presto release on arm64

Error message: 
```
Caused by: java.lang.RuntimeException: library not found: /nativelib/Linux-aarch64/libhadoop.so
	at com.facebook.presto.hadoop.HadoopNative.loadLibrary(HadoopNative.java:88)
```

To fix this, we need to release a new version of https://github.com/prestodb/presto-hadoop-apache2 and upgrade the version in Presto. Latest changes in presto-hadoop-apache2 have added the native libs required for Linux and MacOS aarch64.

--- a/pom.xml
+++ b/pom.xml
@@ -768,7 +768,7 @@
             <dependency>
                 <groupId>com.facebook.presto.hadoop</groupId>
                 <artifactId>hadoop-apache2</artifactId>
-                <version>2.7.4-9</version>
+                <version>2.7.4-12</version>
             </dependency>
 
             <dependency>

